react-challenge.html

```diff
@@ -21,9 +21,10 @@
     <div id="react-container"></div>
     <script type="text/babel">
     // No need to change anything ABOVE this line ^
+    // I did see that line 46 isn't valid and I've tried a few things and I don't understand Why
+    // it doesn't go through, my best guess is its a sytax error but I don't see why I am missing.
```

Comment:
Theres multiple issues with this page. I would encourage you to take a look over this again to find those. 

Hint: The number of issues with the page is 3.